neopixel_spi.py

```diff
@@ -91,45 +81,20 @@ class NeoPixel_SPI(NeoPixel):
         pixels = neopixel_spi.NeoPixel_SPI(board.SPI(), 10)
         pixels.fill(0xff0000)
     """
-    #pylint: disable=invalid-name, super-init-not-called, too-many-instance-attributes
+    #pylint: disable=invalid-name
```

Comment:
What name is invalid?

Reply:
i think it was `self.n` and maybe some others, too few characters or something. i'll remove it and see if it's still complaining.

Reply:
it doesn't like `RESET`

Reply:
changed it. will let linter have one for a change.